core/graph/src/main/java/uk/gov/gchq/gaffer/graph/GraphConfig.java

```diff
@@ -85,6 +86,14 @@ public void setLibrary(final GraphLibrary library) {
         this.library = library;
     }
 
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(final String description) {
+        this.description = description;
```

Comment:
Do we need to enforce any restrictions on the characters that are allowed in the description field?